admin_guide/manage_scc.adoc

```diff
@@ -208,6 +195,77 @@ In order to preserve customized SCCs during upgrades, do not edit settings on
 the default SCCs other than priority, users, and groups.
 ====
 
+[[example-security-context-constraints]]
+=== Example Security Context Constraints Settings
+
+.Without Explicit runAsUser Setting
```

Comment:
The issue and examples are all around Pod's `SecurityContext` and hence I think it's better to move this chapter to user guide. We still need to mention how SCC could affect it and also says that admin should grant appropriate permissions.

Reply:
@php-coder When you say "user guide" do you mean this section? https://docs.openshift.com/container-platform/3.7/architecture/additional_concepts/authorization.html#security-context-constraints
This particular topic focuses on conceptual information.

Reply:
> When you say "user guide" do you mean this section? 

I meant "any document under `dev_guide` directory" (in contrast to `admin_guide` that is dedicated to admins).

> https://docs.openshift.com/container-platform/3.7/architecture/additional_concepts/authorization.html#security-context-constraints
This particular topic focuses on conceptual information.

Can we create a new topic then? It should be somewhere near how to create a pod.

Reply:
@aheslin Do you have any thoughts on where this content should be placed?